### PR TITLE
fixing bug CAT-1097

### PR DESCRIPTION
**Pull request by Achilles-96**
Squashing the commits. I hope I did that correct.

### CODE REVIEW

**Comment by Achilles-96** [comment]
I ran reformat-code, made changes and squashed my commits into one.
Thanks for the help.
When I ran reformat code some of older non-formatted lines are alse modified.
I hope it's for good.
I think now my PR is proper. Please review

**Comment by Achilles-96** [comment]
Please verify this patch.

**Comment by xorrizon** [comment]
Ok to test

File: catroidTest/src/org/catrobat/catroid/uitest/formulaeditor/FormulaEditorEditTextTest.java [hunk 1]
```diff
@@ -133,6 +133,14 @@ public void testSingleTapOnFunctionName() {
 				.getSelectionEnd());
 	}
 
+	public void testHighlightColor() {
+		BackgroundColorSpan editTextColorSpanActual = new BackgroundColorSpan(0xff33b5e5);
+		BackgroundColorSpan editTextColorSpanPresent = (BackgroundColorSpan) Reflection.getPrivateField(
+				new FormulaEditorEditText(getActivity()), "COLOR_HIGHLIGHT");
+		assertTrue("Highlight color is wrong", editTextColorSpanActual.getBackgroundColor() == editTextColorSpanPresent.getBackgroundColor());
+
```
Comment (obusher) [inline catroidTest/src/org/catrobat/catroid/uitest/formulaeditor/FormulaEditorEditTextTest.java]: Please remove the newline. ;)

**Comment by obusher** [comment]
It would be nice if you would not touch (reformat) existing code, because it is semantically neutral.

**Comment by Achilles-96** [comment]
So, should I commit once again

**Comment by obusher** [comment]
Yes. ;)

**Comment by Achilles-96** [comment]
Please merge this bug fix.

**Comment by avihric** [comment]
@obusher Can you review this pls

**Comment by obusher** [comment]
@avihric Everything ok! :)

**Comment by Achilles-96** [comment]
Please merge this fix.

**Comment by avihric** [comment]
I have to clarify some issues with our Jenkins admins, which will happen tomorrow. After that I will merge this. Regards.

**Comment by avihric** [comment]
retest this please

**Comment by Achilles-96** [comment]
Thanks

File: catroidTest/src/org/catrobat/catroid/uitest/formulaeditor/FormulaEditorEditTextTest.java [hunk 2]
```diff
@@ -133,6 +133,13 @@ public void testSingleTapOnFunctionName() {
 				.getSelectionEnd());
 	}
 
+	public void testHighlightColor() {
+		BackgroundColorSpan editTextColorSpanActual = new BackgroundColorSpan(0xff33b5e5);
+		BackgroundColorSpan editTextColorSpanPresent = (BackgroundColorSpan) Reflection.getPrivateField(
+				new FormulaEditorEditText(getActivity()), "COLOR_HIGHLIGHT");
+		assertTrue("Highlight color is wrong", editTextColorSpanActual.getBackgroundColor() == editTextColorSpanPresent.getBackgroundColor());
```
Comment (robertpainsi) [inline catroidTest/src/org/catrobat/catroid/uitest/formulaeditor/FormulaEditorEditTextTest.java]: Just a quick note
If
```assertEquals("...",  editTextColorSpanActual, editTextColorSpanPresent);```
does not work, please use
``` assertSame("...", editTextColorSpanActual.getBackgroundColor(), editTextColorSpanPresent.getBackgroundColor()```

**Comment by robertpainsi** [comment]
As mentioned in the jira ticket, the PR code is fine.
But since it's an UXI change, the UXI-Team also has to review the changes.

If they say it's good to merge, it'll be merged immediately! :smirk_cat:

**Comment by avihric** [comment]
@robertpainsi Please wait with merging anything until Monday.

**Comment by robertpainsi** [comment]
Sir! Yes, Sir! :guardsman:

**Comment by linsm** [comment]
please reopen your pull request

use your forked repository
target of pull request should be 'develop' branch
link THIS pull request in the new one

**Pull request closed without merging**